### PR TITLE
HPCDB - modw etlv2 configuration files

### PR DESCRIPTION
**Pull request by plessbd**
This is the configuration files needed for the hpcdb - modw etlv2 code changes.
This is low impact as nothing should run these pipelines or actions.  the only risk is parsing errors that could arise from other etl v2 processes.

This is to make code review easier.  This will be 1 of 3 different pull requests replacing #200 
1) config files
2) code cleanup
3) actual code changes and additions

### CODE REVIEW

File: configuration/etl/etl_tables.d/xdmod/account.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+    "table_definition": {
+        "name": "account",
+        "engine": "MyISAM",
+        "comment": "This table has records for all the TeraGrid accounts.",
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/account.json]: Lets change this to "This table has records for all the accounts/projects."

File: configuration/etl/etl_tables.d/xdmod/account.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                "type": "int(11)",
+                "nullable": true
+            }
+        ],
+        "indexes": [{
+                "name": "PRIMARY",
+                "columns": [
+                    "id"
+                ],
+                "type": "BTREE",
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/account.json]: You technically don't need the type unless you're changing it but it's fine to leave it in.

File: configuration/etl/etl_tables.d/xdmod/error-descriptions.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+    "table_definition": {
+        "comment": "",
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/error-descriptions.json]: Don't need to include empty comments. Fill in or remove.

File: configuration/etl/etl_tables.d/xdmod/error-descriptions.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        "engine": "MyISAM",
+        "columns": [{
+                "type": "int(11)",
+                "nullable": false,
+                "default": null,
+                "extra": null,
+                "comment": null,
+                "hints": null,
+                "rename": null,
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/error-descriptions.json]: You shouldn't need `null` values for extra, comment, hints, or rename.

File: configuration/etl/etl_tables.d/xdmod/error-descriptions.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+                "name": "id"
+            },
+            {
+                "type": "varchar(255)",
+                "nullable": false,
+                "default": null,
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/error-descriptions.json]: Same here.

File: configuration/etl/etl_tables.d/xdmod/min-max-date.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
+{
+    "table_definition": {
+        "comment": "",
+        "engine": "InnoDB",
+        "columns": [{
+                "type": "datetime",
+                "nullable": true,
+                "default": null,
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/min-max-date.json]: You can clean these up too.

File: configuration/etl/etl_tables.d/xdmod/organization.json [hunk 7]
```diff
@@ -0,0 +1,160 @@
+{
+    "table_definition": {
+        "name": "organization",
+        "engine": "MyISAM",
+        "comment": "The various organization pertaining to TG.",
```
Comment (smgallo) [inline configuration/etl/etl_tables.d/xdmod/organization.json]: Change to a non-TG specific comment.

File: configuration/etl/etl_tables.d/xdmod/error-descriptions.json (hunk 3 above)
Comment (plessbd) [reply configuration/etl/etl_tables.d/xdmod/error-descriptions.json]: if the table manager didnt dump bad data then I wouldn't include it..  **cough** your code **cough**

Comment (smgallo) [reply configuration/etl/etl_tables.d/xdmod/error-descriptions.json]: So my code is bad and I don't feel bad. Humpf.

**Comment by plessbd** [comment]
Will add comments after I determine where the code should go...

File: configuration/etl/etl_sql.d/hpcdb-modw/jobhosts-post-ingest-truncate.sql [hunk 8]
```diff
@@ -0,0 +1,3 @@
+-- Truncate the staging table
```
[the remark this answers is not in the document]
Comment (plessbd) [reply configuration/etl/etl_sql.d/hpcdb-modw/jobhosts-post-ingest-truncate.sql]: I like to clean up after it is done.  I guess it technically doesnt need to be and could be done at the beginning, but I prefer not to leave "useless" data sitting around

**Pull request merged**